src/partisan/icommands.py

```diff
@@ -95,6 +97,18 @@ def irm(remote_path: Union[PurePath, str], force=False, recurse=False):
         cmd.append("-r")
 
     cmd.append(remote_path)
+
+    # Workround for iRODS 4.2.7 which insists on raising errors with when the target is
+    # absent, even when -f is used. This should be silent for a missing target.
+    try:
+        _run(cmd)
+    except RodsError:
+        if not force:
+            raise
```

Comment:
worth doing a:
```suggestion
            raise
          else:
            log.warning(f"iRODS exception raised when executing {cmd}")
```

Just because silently ignoring exceptions makes me worried. Also I guess RodsError is a more broader error than just target is absent. If not `warning` maybe `info` or `debug`?